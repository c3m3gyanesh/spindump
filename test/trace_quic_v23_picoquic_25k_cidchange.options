--format text --report-spins --report-notes