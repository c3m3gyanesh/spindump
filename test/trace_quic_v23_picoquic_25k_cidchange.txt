Picoquic, with a change of CIDs in the middle of the connection
