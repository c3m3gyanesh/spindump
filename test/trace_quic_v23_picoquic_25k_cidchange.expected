QUIC 172.16.95.160 <-> 34.238.92.221 083cdbd227d37fc9-3e2c644db5d0d2fb (53797:4433) at 1571090228901590 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 083cdbd227d37fc9-08681aea136baa43 (53797:4433) at 1571090228984461 measurement up right 82871 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090228985133 spinvalue up 0 responder packets 1 3 bytes 1280 3822 bandwidth 1280 3822 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090228985729 spinvalue up 1 initiator packets 2 4 bytes 1707 5290 bandwidth 1707 5290 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090228985730 spinvalue up 1 initiator packets 3 4 bytes 1799 5290 bandwidth 1799 5290 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229064738 spinvalue up 1 responder packets 4 5 bytes 3279 5423 bandwidth 3279 5423 note "V23,no spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229064750 spinvalue up 0 initiator packets 4 6 bytes 3279 6891 bandwidth 3279 6891 note "V23,no I-spin"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229064750 measurement up left 12 packets 4 6 bytes 3279 6891 bandwidth 3279 6891 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229064895 spinvalue up 0 initiator packets 5 6 bytes 3362 6891 bandwidth 3362 6891 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229065711 spinvalue up 1 responder packets 6 6 bytes 3445 6891 bandwidth 3445 6891 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229065713 spinvalue up 1 responder packets 6 7 bytes 3445 8359 bandwidth 3445 8359 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229065857 spinvalue up 0 initiator packets 6 8 bytes 3445 9827 bandwidth 3445 9827 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229070690 spinvalue up 1 responder packets 7 8 bytes 3528 9827 bandwidth 3528 9827 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229070692 spinvalue up 1 responder packets 7 9 bytes 3528 11295 bandwidth 3528 11295 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229070866 spinvalue up 0 initiator packets 7 10 bytes 3528 12763 bandwidth 3528 12763 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229077087 spinvalue up 1 responder packets 8 10 bytes 3611 12763 bandwidth 3611 12763 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229077094 spinvalue up 1 responder packets 8 11 bytes 3611 14231 bandwidth 3611 14231 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229077476 spinvalue up 0 initiator packets 8 12 bytes 3611 15699 bandwidth 3611 15699 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229083037 spinvalue up 1 responder packets 9 12 bytes 3694 15699 bandwidth 3694 15699 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229083744 spinvalue up 1 responder packets 9 13 bytes 3694 17167 bandwidth 3694 17167 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229084068 spinvalue up 0 initiator packets 9 14 bytes 3694 18635 bandwidth 3694 18635 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229088886 spinvalue up 1 responder packets 10 14 bytes 3777 18635 bandwidth 3777 18635 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229088894 spinvalue up 1 responder packets 10 15 bytes 3777 20103 bandwidth 3777 20103 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229089204 spinvalue up 0 initiator packets 10 16 bytes 3777 21571 bandwidth 3777 21571 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229094819 spinvalue up 1 responder packets 11 16 bytes 3860 21571 bandwidth 3860 21571 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229094826 spinvalue up 1 responder packets 11 17 bytes 3860 23039 bandwidth 3860 23039 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229095216 spinvalue up 0 initiator packets 11 18 bytes 3860 24507 bandwidth 3860 24507 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229100638 spinvalue up 1 responder packets 12 18 bytes 3943 24507 bandwidth 3943 24507 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229100645 spinvalue up 1 responder packets 12 19 bytes 3943 25975 bandwidth 3943 25975 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229100967 spinvalue up 0 initiator packets 12 20 bytes 3943 27443 bandwidth 3943 27443 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229143771 spinvalue up 0 responder packets 13 20 bytes 4026 27443 bandwidth 4026 27443 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229143771 measurement up right 79021 packets 13 20 bytes 4026 27443 bandwidth 4026 27443 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229143771 measurement up full (responder) 79033 packets 13 20 bytes 4026 27443 bandwidth 4026 27443 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229143778 spinvalue up 1 responder packets 13 21 bytes 4026 27514 bandwidth 4026 27514 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229143778 measurement up full (responder) 7 packets 13 21 bytes 4026 27514 bandwidth 4026 27514 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144087 spinvalue up 1 responder packets 13 22 bytes 4026 28982 bandwidth 4026 28982 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144096 spinvalue up 1 responder packets 13 23 bytes 4026 30162 bandwidth 4026 30162 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144162 spinvalue up 1 initiator packets 13 24 bytes 4026 31630 bandwidth 4026 31630 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144162 measurement up left 391 packets 13 24 bytes 4026 31630 bandwidth 4026 31630 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144162 measurement up full (initiator) 79412 packets 13 24 bytes 4026 31630 bandwidth 4026 31630 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229144790 spinvalue up 1 initiator packets 14 24 bytes 4109 31630 bandwidth 4109 31630 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229221642 spinvalue up 1 responder packets 15 24 bytes 4174 31630 bandwidth 4174 31630 note "V23,spinning"
QUIC 172.16.95.160 <-> 34.238.92.221 3cdbd227d37fc9d4-08681aea136baa43 (53797:4433) at 1571090229222176 spinvalue up 1 responder packets 15 25 bytes 4174 31691 bandwidth 4174 31691 note "V23,spinning"
